### PR TITLE
support google stt version 2

### PR DESCRIPTION
**Pull request by xquanluu**

### CODE REVIEW

**Comment by davehorton** [comment]
please add support for this:
[voice activity timeout params](https://cloud.google.com/speech-to-text/v2/docs/reference/rpc/google.cloud.speech.v2#voiceactivitytimeout)

**Comment by davehorton** [comment]
also [transcript normalization](https://cloud.google.com/speech-to-text/v2/docs/reference/rpc/google.cloud.speech.v2#transcriptnormalization)

**Comment by xquanluu** [comment]
fixed all review comments above

File: specs.json [hunk 1]
```diff
@@ -698,6 +699,25 @@
...
+  "googleOptions" : {
+    "properties": {
+      "serviceVersion": {
+        "type": "string",
+        "enum": [
+          "v1",
+          "v2"
+        ]
+      },
+      "voiceActivityTimeout": {
```
Comment (davehorton) [inline specs.json:711]: hmm, no, we need the option to separately set each one, so as far as the application user is concerned there should be `speechStartTimeoutMs` and `speechEndTimeoutMs`

File: specs.json [hunk 2]
```diff
@@ -698,6 +699,25 @@
...
+        "type": "string",
+        "enum": [
+          "speech_start_timeout",
+          "speech_end_timeout"
+        ]
+      },
+      "transcriptNormalization": "array"
```
Comment (davehorton) [inline specs.json:718]: Yes this is good.  As far as what the user will provide it should actually be an array of arrays, e.g
```
[['dog', 'cat'], ['damn it', 'darn it']]
```

**Review by davehorton — CHANGES_REQUESTED** [review]

File: specs.json (hunk 1 above)
Comment (xquanluu) [reply specs.json]: my bad, updated

File: specs.json (hunk 2 above)
Comment (xquanluu) [reply specs.json:713]: yes

**Review by davehorton — APPROVED** [review]

**Pull request merged**